packages/generators/src/handlers/default.ts

```diff
@@ -198,7 +192,6 @@ export function generate(self: CustomGenerator): void {
     ...self.answers,
     entry,
   });
-  self.configurationPath = self.destinationPath("webpack.config.js");
```

Comment:
Not a big deal for now, but IMO, This should be handled here only, as templates might generate multiple configuration files as well in future.

Reply:
Yep, let's do refactor when we will implement it, I think now it good, focus on types for webpack-cli